admin/app/jobs/R2PagePressJob.scala

```diff
@@ -85,11 +83,11 @@ object R2PagePressJob extends ExecutionContexts with Logging {
   private def parseAndClean(originalDocSource: String): Future[String] = {
     val cleaners = Seq(PollsHtmlCleaner, InteractiveHtmlCleaner, SimpleHtmlCleaner)
     val archiveDocument = Jsoup.parse(originalDocSource)
-    cleaners.filter(_.canClean(archiveDocument))
+    val doc = cleaners.filter(_.canClean(archiveDocument))
```

Comment:
may be worth adding a type on doc, as otherwise it will always compile even if we make it stop returning a Document by mistake, as toString always exists on anything (which is the main problem with toString, but we can't easily get around that)